python/cudf/cudf/_lib/utils.pyx

```diff
@@ -153,7 +154,18 @@ cpdef generate_pandas_metadata(Table table, index):
     md_dict = json.loads(metadata[b"pandas"])
 
     # correct metadata for list and struct types
+    type_map = {
+        str(cudf_dtype): str(pandas_dtype)
+        for cudf_dtype, pandas_dtype in cudf_dtypes_to_pandas_dtypes.items()
+    }
+
```

Comment:
```suggestion
    PARQUET_META_TYPE_MAP = {
        str(cudf_dtype): str(pandas_dtype)
        for cudf_dtype, pandas_dtype in cudf_dtypes_to_pandas_dtypes.items()
    }
```
Can we move this map creation to the top of this file outside this function? That way we don't need to create this map everytime this method is called. Also, rename it to `PARQUET_META_TYPE_MAP`?(All capitals since this would be globally accesible outside this method)